reportek/core/permissions/utils.py

```diff
@@ -69,9 +71,25 @@ def debug_call(f):
     Wrapper for `has_permissions` | `has_object_permissions`,
     logs the call and result.
     """
+
     @wraps(f)
     def wrapper(self, request, view, *args, **kwargs):
         r = f(self, request, view, *args, **kwargs)
-        debug(f'{request.method} [{view.action}] in {self.__class__.__name__}.{f.__name__}() -> {r}')
+        debug(
+            f'{request.method} [{view.action}] in {self.__class__.__name__}.{f.__name__}() -> {r}'
+        )
         return r
+
+    return wrapper
+
+
+def skip_for_superuser(f):
+    """
+    Wrapper for `has_permissions` | `has_object_permissions`, skips checks for superusers.ß
```

Comment:
Errrm what does the ß do? :)

Reply:
Hopefully nothing, I hit Alt-s instead of Cmd-s :)